lib/html/pipeline/dradis_escape_html_filter.rb

```diff
@@ -13,18 +13,24 @@ class Pipeline
     #
     # This filter does not write any additional information to the context hash.
     class DradisEscapeHTMLFilter < TextFilter
+      PROTOCOLS = %w{
+        ed2k ftp http https irc mailto news gopher nntp telnet webcal xmpp
+        callto feed svn urn aim rsync tag ssh sftp rtsp afs file
+      }
+
       def call
         text = ERB::Util.html_escape(@text)
 
-        # Match the text between bc. and bc.. following the textile rules
+        # Match the text under bc./bc.. and links, following the textile rules
         regex = Regexp.union(
-          /(?<=bc. )(.*?)(?=(\r\n|\n){2})/m,
-          /(?<=bc.. )(.*?)(?=(bc\.|bc\.\.|p\.|\z))/m
+          /(?<=bc\. )(.*?)(?=(\r\n|\n){2})/m,
+          /(?<=bc\.\. )(.*?)(?=(bc\.|bc\.\.|p\.|\z))/m,
+          /&quot;.*&quot;:(?:#{PROTOCOLS.join('|')})\:\/\/.+/
```

Comment:
- we probably need to use the `(.*?)` formula, no?
- don't we need the `/m`?
- see above re: hard-coding the supported protocols here

Reply:
I don't think we'd need the `/m` since there can be no multiline links.